tests/utils/pvc.go

```diff
@@ -256,3 +258,55 @@ func WaitPVCDeletedByUID(clientSet *kubernetes.Clientset, pvcSpec *k8sv1.Persist
 	})
 	return result, err
 }
+
+func getCdiCR(clientSet *cdiClientset.Clientset) (*cdiv1.CDI, error) {
+	crList, err := clientSet.CdiV1beta1().CDIs().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	if len(crList.Items) != 1 {
+		return nil, fmt.Errorf("There should be a single Cdi, %d items found", len(crList.Items))
+	}
+	return &crList.Items[0], nil
+}
+
+// WaitForCDICrCloneStrategy waits for a CDI CR Clone strategy
+func WaitForCDICrCloneStrategy(clientSet *cdiClientset.Clientset, cloneStrategy cdiv1.CDICloneStrategy) error {
+	err := wait.PollImmediate(pvcPollInterval, pvcCreateTime, func() (bool, error) {
+		var err error
+		cr, err := getCdiCR(clientSet)
+		if err != nil {
+			if apierrs.IsNotFound(err) {
+				return false, nil
+			}
+			return false, err
+		}
+		if cr.Spec.CloneStrategyOverride != nil && cloneStrategy == *cr.Spec.CloneStrategyOverride {
```

Comment:
This func and the following one are similar except this check which can be passed as a func parameter, so a refactor will be nice.